aana/core/models/ray.py

```diff
@@ -0,0 +1,48 @@
+from pydantic import BaseModel, Field
+
+
+class Resources(BaseModel):
+    """Resources of a video.
+
+    Attributes:
+        CPU (float): Available CPU.
+        GPU (float): Available GPU.
+        memory (float): Available Ram.
+    """
+
+    CPU: float = Field(0, description="Available CPU.")
+    GPU: float = Field(0, description="Available GPU.")
+    memory: float = Field(0, description="Available Ram.")
+
+    def __add__(self, other: "Resources") -> "Resources":
+        """Sum two Resources."""
+        return Resources(
+            CPU=self.CPU + other.CPU,
+            GPU=self.GPU + other.GPU,
+            memory=self.memory + other.memory
```

Comment:
The default ray resources keys are like these. (GPU, CPU, memory). But I think we can capitalize the memory too!
Let me know